### ISSUE TITLE
Permettre la répétition d'<edition> dans <editionStmt>

### ISSUE THREAD

**Issue by laurentromary** [CCSDForge/HAL#22]
Important pour refléter les version dans HAL.

**Comment by laurentromary** [comment]
En fait, en creusant, il serait nécessaire de répéter `<editionStmt>` pour chaque édition. Ca permettrait de pouvoir, par exemple, avoir des contributeurs différents à chaque version. @ybarborini @marmol vous en pensez quoi? Ca se laisse changer facilement?

**Comment by ybarborini** [comment]
C'est déjà le cas actuellement dans HAL non ?
Si tu prends l'exemple https://hal.archives-ouvertes.fr/hal-00000001/tei 
il y a bien une répétition de edition dand editionStmt

**Comment by laurentromary** [comment]
Regarde mon deuxième commentaire (il y avait une balise que je n'avais pas échappé...), c'est en fait une erreur et il faudrait répéter l'élément parent. Ca irait?

**Comment by ybarborini** [comment]
OK. 
```
<editionStmt>
  <edition n="v1">
    <date type="whenSubmitted">2002-09-23 20:01:38</date>
    ...
  </edition>
  <edition n="v2" type="current">
    <date type="whenSubmitted">2004-11-14 12:50:42</date>
    ...
  </edition>
</editionStmt>
```
serait remplacé par 
```
<editionStmt>
  <edition n="v1">
    <date type="whenSubmitted">2002-09-23 20:01:38</date>
    ...
  </edition>
</editionStmt>
<editionStmt>
  <edition n="v2" type="current">
    <date type="whenSubmitted">2004-11-14 12:50:42</date>
    ...
  </edition>
</editionStmt>
```

**Comment by laurentromary** [comment]
Exactement. Ca permet de mettre d'autres informations dans `<editionStmt>` qui pourraient être différente entre ces blocs: par exemple la personne responsable => actuellement, on ne sait pas décrire qui a soumis et qui a modifié un dépôt.

**Comment by laurentromary** [comment]
Comment on procède pour l'implémentation si vous êtes d'accord. Tu signales la mise à jour, tu produis un exemple et j'implémente dans le schéma?

**Comment by ybarborini** [comment]
On peut en effet faire comme ça. je signale dès que c'est en place en preprod en fournissant un exemple
